src/evergreen/api.py

```diff
@@ -842,6 +842,16 @@ def single_test_by_task_and_test_file(self, task_id: str, test_file: str) -> Lis
         param = {"test_name": test_file}
         return [Tst(test, self) for test in self._call_api(url, params=param).json()]
 
+    def num_of_tests_by_task(self, task_id: str) -> int:
```

Comment:
[nit] It might be nice to add a helper method on the `Task` object [here](https://github.com/evergreen-ci/evergreen.py/blob/aef69230ddc64bab1498325f6fd9e2dbdf4be518/src/evergreen/task.py#L101) that will call this function for the given task. That would provide a nicer workflow because I imagine we will frequently have a task object when we want to call this:

```python
evg_task = evg_api.task_by_id(task_id)
if evg_task.get_num_test() < 100:
   # do something
```

Reply:
I think he added a helper function [here](https://github.com/evergreen-ci/evergreen.py/pull/122/files#diff-7d55acd293f6c17f97af65665554935c42e1d1d5488ef233a6953f3fb1867ea1R342) already if I'm not mistaken?

Reply:
Yeah, it's already there.